components/Tip.tsx

```diff
@@ -0,0 +1,84 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import {
+  Box,
+  Typography,
+  Button,
+  Paper,
+  Stack,
+  IconButton,
+  useTheme,
+} from '@mui/material';
+import { Lightbulb, ChevronDown } from 'react-bootstrap-icons';
+
+interface TipProps {
+  title: string;
+  text: string;
+  storageKey: string;
+}
+
+export const Tip = ({ title, text, storageKey }: TipProps) => {
+  const theme = useTheme();
+  const [isTextVisible, setIsTextVisible] = useState(true);
+  const [isTipOpen, setIsTipOpen] = useState(true);
+
+  useEffect(() => {
+    const savedTipState = localStorage.getItem(storageKey);
+    if (savedTipState === 'closed') {
+      setIsTextVisible(false);
+    }
+  }, [storageKey]);
+
+  const handleDismiss = () => {
+    setIsTextVisible(false);
+    setIsTipOpen(false);
+    localStorage.setItem(storageKey, 'closed');
+  };
+
+  const handleToggleTip = () => {
+    const newTipState = !isTextVisible;
+    setIsTextVisible(newTipState);
+    localStorage.setItem(storageKey, newTipState ? 'open' : 'closed');
+  };
+
+  return (
+    <Paper
```

Comment:
Could we use the mui `Accordion` component for this? That would make it consistent with other accordions where the whole title is clickable and there's a nice transition between open/ closed